econml/tests/monte_carlo_statsmodels.py

```diff
@@ -0,0 +1,298 @@
+import numpy as np

```

Comment:
Is the test folder the right folder to add monte carlo tests? These are not unit tests and take a lot of time.

Reply:
Removed as discussed.